cmake/Modules/FindMKL.cmake

```diff
@@ -302,14 +303,17 @@ if (USE_MKL AND USE_IDEEP)
   endif()
   
   if (MKLDNN_INCLUDE_DIR)
+    list(APPEND IDEEP_INCLUDE_DIR ${MKLDNN_INCLUDE_DIR})
+    list(APPEND __ideep_looked_fo ${MKLDNN_INCLUDE_DIR})
```

Comment:
I think this should be `__ideep_looked_for`?

Reply:
Good catch.